e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js

```diff
@@ -270,6 +270,36 @@ describe('Leave an archived channel', () => {
 
         // * Private archived channels you are not a member above are not available on channel switcher
         cy.contains('#suggestionList', otherChannelName).should('not.exist');
+        cy.get('#quickSwitchModalLabel button.close').click();
+    });
+    it('MM-T1677 Archived channels are not shown as unread in channel switcher', () => {
+        // # As the test user join a public channel then open any other channel in the drawer
+        cy.uiCreateChannel('arhived-not-read').then(({name}) => {
+            cy.visit(`/${testTeam.name}/channels/off-topic`);
+
+            // # As another user post in the channel from step 1. then archive it
+            cy.apiLogout();
+            cy.get('#loginButton').should('be.visible');
+            cy.apiLogin(otherUser);
```

Comment:
I'm getting intermittent failed test on `loginButton` visibility check. It's probably due varying timing in firing out session redirect to login page. If you're going to test visibility of `loginButton`, then I'd suggest to do logout via UI flow and not thru API command. Otherwise, you may entirely remove it since we're actually not testing logging out then logging in UI flow.
```suggestion
            cy.apiLogout();
            cy.apiLogin(otherUser);
```